src/modules/ellswift/main_impl.h

```diff
@@ -462,6 +462,7 @@ int secp256k1_ellswift_create(const secp256k1_context *ctx, unsigned char *ell64
 
     /* Compute (affine) public key */
     ret = secp256k1_ec_pubkey_create_helper(&ctx->ecmult_gen_ctx, &seckey_scalar, &p, seckey32);
+    secp256k1_declassify(ctx, &p, sizeof(p)); /* not constant time in produced pubkey */
```

Comment:
70d80ae: why don't variables in the `ellswift_xdh` code which interact with private key not use `secp256k1_declassify`? i don't understand the context for what kind of memory needs to be marked as not secret here.

Reply:
thanks for the detailed explanation! understood when declassify needs to be used now.